src/blocks/card.css

```diff
@@ -0,0 +1,90 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  max-width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  text-align: center;
+  margin-bottom: 36px;
+}
+
+.card__img-product {
+  height: 134px;
+}
+
+.card__content {
+  padding: 32px 16px 16px;
+}
+
+.card__title {
+  margin-bottom: 4px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.card__code {
+  display: block;
+  margin-bottom: 16px;
+  color: #616070;
+}
+
+.card__mark {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.card__mark-comment {
+  align-items: flex-end;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 16px;
+
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.card__price-teg {
+  font-size: 16px;
+  font-weight: 700;
+  color: #060b35;
+}
+
+.card__btn {
+  display: block;
+  padding: 11px 54px;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  text-decoration: none;
+
+  border-radius: 5px;
+  border: 1px solid #00acdc;
```

Comment:
You could use there `transparent` border color, because it the same as background(line 84).